docusaurus/docs/Android/04-compose/03-channel-components/01-channels-screen.mdx

```diff
@@ -93,11 +95,11 @@ fun ChannelsScreen(
         Filters.eq("type", "messaging"),
         Filters.`in`("members", listOf(ChatClient.instance().getCurrentUser()?.id ?: ""))
     ),
-    querySort: QuerySort<Channel> = QuerySort.Companion.desc("id"),
+    querySort: QuerySort<Channel> = QuerySort.desc("last_updated"),
```

Comment:
This will work, but the `Id` field is not indexed. According to the docs:

```
You can sort based on last_updated, last_message_at, updated_at, created_at, member_count, unread_count or has_unread(unread status)
```